lib/shanty_properties_file_plugin.rb

```diff
@@ -0,0 +1,3 @@
+# Public: shanty properties file plugin module
```

Comment:
This file should require the other files in your gem that you want to load when somebody does `require 'shanty_properties_file_plugin'`.